pkg/volume/aws_ebs/aws_ebs_test.go

```diff
@@ -118,9 +135,9 @@ func TestPlugin(t *testing.T) {
 	if builder == nil {
 		t.Errorf("Got a nil Builder")
 	}
-
+	volPath := path.Join(tmpDir, "pods/poduid/volumes/kubernetes.io~aws-ebs/vol1")
```

Comment:
Doesn't require a fix, just FYI:

On https://github.com/jijun2/kubernetes/blob/fix-cni_test/pkg/kubelet/network/cni/cni_test.go#L172 you split up into chunks. Just be aware you are sometimes doing that and sometimes joining a path with a forward slash'd path.